addons/website_forum/models/forum.py

```diff
@@ -865,6 +865,7 @@ class PostReason(models.Model):
 class Vote(models.Model):
     _name = 'forum.post.vote'
     _description = 'Post Vote'
+    _order = 'create_date desc'
```

Comment:
Maybe also add `, id desc` as tie-breaker to avoid any potential non deterministic issue?

Reply:
Risk is really weak, but as suggested by seb it is a good practice on new order line to add it if it could potentially happens.

(New line = don’t make a PR to add it everywhere 😝 if it is existing and have no know impact we don’t add it)

It is case by case since that could have an impact to the index key of psql ...

Reply:
@seb-odoo and @JKE-be thanks for the feedback. I don't get what the difference is between ordering by just `create_date` or by `create_date, id`. What issue could it give to just order by `create_date`?

Reply:
`create_date` is not unique, so two models can potentially have the same value.

In that case, the order between those two will be "random" (depends on orm, psql, ...).

We don't want to create code that is not deterministic like that: it could cause false positives on tests, it could flicker the interface on refresh, and at worse (in some cases but not here) it could create business issues when the order is important.

Reply:
Gotcha, fixed it.